src/OSPSuite.Presentation/OSPSuite.Presentation.csproj

```diff
@@ -31,6 +31,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="DevExpress.Data" Version="19.2.4" />
```

Comment:
@Yuri05 Devexpress has created a package just with this dll thus replacing effectively our own OSPSuite.DevExpress.Presentation